front/tests/pages/stdcm-page-model.ts

```diff
@@ -294,15 +325,68 @@ class STDCMPage {
   }
 
   // Fills fields with test values in the consist section
-  async fillConsistDetails() {
-    await this.tractionEngineField.fill(electricRollingStockName);
-    await this.tractionEngineField.press('ArrowDown');
-    await this.tractionEngineField.press('Enter');
-    await this.tractionEngineField.dispatchEvent('blur');
-    await this.tonnageField.fill('400');
-    await this.lengthField.fill('300');
-    await this.codeCompoField.selectOption('HLP');
-    await this.maxSpeedField.fill('180');
+  async fillAndVerifyConsistDetails(
+    fields: ConsistFields,
+    prefilledTractionEngineTonnage: string,
+    prefilledTractionEngineLength: string,
+    prefilledTractionEngineMaxSpeed: string,
+    prefilledTowedRollingStockTonnage?: string,
+    prefilledTowedRollingStockLength?: string,
+    prefilledTowedRollingStockMaxSpeed?: string
+  ): Promise<void> {
```

Comment:
```suggestion
  async fillAndVerifyConsistDetails(
    fields: ConsistFields,
    prefilledTractionEngineTonnage: string,
    prefilledTractionEngineLength: string,
    prefilledTractionEngineMaxSpeed: string,
    prefilledTowedRollingStockTonnage?: string,
    prefilledTowedRollingStockLength?: string,
    prefilledTowedRollingStockMaxSpeed?: string
  ): Promise<void> {
  
    const { tractionEngine, towedRollingStock, tonnage, length, maxSpeed, speedLimitTag } = fields;

```

then you can remove all `fields.`

Reply:
Done